README.md

```diff
@@ -16,8 +16,13 @@ the square brackets (for multiple-choice questions).
 What does it mean for an HTML tag to be "semantic"? Give an example of a short snippet of HTML written using semantic tags and non-semantic tags.
 
 ```text
-Your answer here.
+A 'semantic' tag uses language/terms that express their intended use for the browser.
 ```
+example HTML:
+```
+<section><p>Here is a paragh to start <span>this section</span> of the page</p></section>
+```
+<section> and <p> are semantic while <span> is not.
 
```

Comment:
Yes, "Semantic" is referring to the Function over the Form or Style of html code